store-rx2/src/test/kotlin/com/dropbox/store/rx2/test/HotRxSingleStoreTest.kt

```diff
@@ -0,0 +1,72 @@
+package com.dropbox.store.rx2.test
+
+import com.dropbox.android.external.store4.ResponseOrigin
+import com.dropbox.android.external.store4.StoreRequest
+import com.dropbox.android.external.store4.StoreResponse
+import com.dropbox.store.rx2.observe
+import com.dropbox.store.rx2.rxSingleStore
+import com.dropbox.store.rx2.withSinglePersister
+import io.reactivex.Maybe
+import io.reactivex.Single
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import java.util.concurrent.atomic.AtomicInteger
+
+@RunWith(JUnit4::class)
+class HotRxSingleStoreTest {
```

Comment:
Can we move the hotness test to store itself? it could be a problem for flow with side-effects as well

Reply:
I can't figure out how to make test fail on that side

Reply:
made `HotFlowStoreTest`